ci-operator/config/openshift/gcp-pd-csi-driver-operator/openshift-gcp-pd-csi-driver-operator-master.yaml

```diff
@@ -63,6 +63,14 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-csi
+- as: e2e-gcp-ovn-upgrade
+  steps:
+    cluster_profile: gcp
+    workflow: openshift-upgrade-gcp-ovn
+- as: e2e-gcp-ovn-rt-upgrade
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    workflow: openshift-upgrade-gcp-ovn-rt
```

Comment:
`-rt-` means realtime kernel. IMO it's not useful to test both non-rt and rt kernel, it'd personally add just the non-rt job.